.github/workflows/tests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - run: scripts-dev/check_schema_delta.py --force-colors
 
   lint:
+    permissions: {}
     uses: "matrix-org/backend-meta/.github/workflows/python-poetry-ci.yml@v1"
     with:
       typechecking-extras: "all"
```

Comment:
Does `permissions: {}` even work here? Doesn't the called workflow need permission to at least checkout the repo?